elements/lisk-p2p/src/p2p.ts

```diff
@@ -358,6 +324,18 @@ export class P2P extends EventEmitter {
 			this.emit(EVENT_CLOSE_INBOUND, closePacket);
 		};
 
+		this._handleInboundPeerConnect = (
+			incomingPeerConnection: IncomingPeerConnection,
+		) => {
+			this._peerPool.addInboundPeer(
```

Comment:
This function throws an error when it already exists in peer pool, we should wrap it around `try/catch` and emit relevant events